.github/CODEOWNERS

```diff
@@ -55,7 +55,7 @@
 /sdk/eventgrid/                                     @srnagar @yijunxiems
 
 # PRLabel: %Event Hubs
-/sdk/eventhubs/                                     @conniey @srnagar @mssfang @yijunxiems
+/sdk/eventhubs/                                     @conniey @srnagar @yijunxiems
```

Comment:
this changes will be disappear after I merge https://github.com/Azure/azure-sdk-for-java/pull/21245